src/pages/adminLogin.jsx

```diff
@@ -9,17 +9,28 @@ import Box from "@mui/material/Box";
 import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
 import Typography from "@mui/material/Typography";
 import Paper from "@mui/material/Paper";
+import { useNavigate } from "react-router-dom";
 import Container from "@mui/material/Container";
 import { ThemeProvider } from "@mui/material/styles";
 import theme from "../theme";
+import { ToastContainer, toast } from "react-toastify";
 export default function AdminLogin() {
+  const navigate = useNavigate();
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
   const handleSubmit = (event) => {
     event.preventDefault();
-    const data = new FormData(event.currentTarget);
-    console.log({
-      email: data.get("email"),
-      password: data.get("password")
-    });
+    if (email === "faisal@luminogics.com" && password === "faisal22") {
+      localStorage.setItem("email", "faisal@luminogics.com");
```

Comment:
If you are using Redux then localStorage should be updated through Redux.

Reply:
data is stored in redux but local storage is used for creating session only because first time when page is rendered the store will be empty

Reply:
Okay

Reply:
Still doesn't really make any sense to me, if app is loaded, redux store is initialized you can set a value in it.